File: patterns/c-patterns/pattern93.c
// Pattern 93: ...

// *       *
// **     **
// ***   ***
// **** ****
// *********

#include <stdio.h>

int main()
{
    int n = 5;

    int i, j;

    for (i = 1; i <= n; i++)
    {
        for (j = 1; j < n * 2; j++)
        {
            if (j <= i || j >= n * 2 - i)
            {
                printf("*");
            }
            else
            {
                printf(" ");
            }
        }
        printf("\n");
    }
    return 0;
}

File: patterns/javascript-patterns/pattern93.js
// Pattern 93: ...

// *       *
// **     **
// ***   ***
// **** ****
// *********

let n = 5;

for (let i = 1; i <= n; i++) {
    let row = '';

    for (let j = 1; j < n * 2; j++) {
        if (j <= i || j >= n * 2 - i) {
            row += '*';
        } else {
            row += ' ';
        }
    }

    console.log(row);
}
